engine/src/main/java/org/terasology/registry/CoreRegistry.java

```diff
@@ -69,4 +71,46 @@ public static void setContext(Context context) {
         return context.get(type);
     }
 
+    /**
+     * Returns this class as a Context object.
+     *
+     * @return A Context object representing this class.
+     */
+    public static Context asContext() {
+        return CoreRegistryContext.INSTANCE;
```

Comment:
This returns a Context that behaves like CoreRegistry - if you change the CoreRegistry backing context, it reflects on the CoreRegistryContext.